<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="JP" failures="7" tests="7" name="Login" time="0.053" errors="0" timestamp="2025-08-18T09:33:32 IST">
  <testcase classname="TestCases.login" name="login_case_1" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.check_validation(LoginPage.java:42)
at TestCases.login.login_case_1(login.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_1 -->
  <testcase classname="TestCases.login" name="login_case_2" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.enter_email_only(LoginPage.java:54)
at TestCases.login.login_case_2(login.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_2 -->
  <testcase classname="TestCases.login" name="login_case_3" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.enter_password_only(LoginPage.java:69)
at TestCases.login.login_case_3(login.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_3 -->
  <testcase classname="TestCases.login" name="login_case_4" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.enter_email_correct_password_wrong(LoginPage.java:115)
at TestCases.login.login_case_4(login.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_4 -->
  <testcase classname="TestCases.login" name="login_case_5" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.enter_email_wrong_password_correct(LoginPage.java:99)
at TestCases.login.login_case_5(login.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_5 -->
  <testcase classname="TestCases.login" name="login_case_6" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pages.LoginPage.enter_email_password_correct(LoginPage.java:131)
at TestCases.login.login_case_6(login.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login_case_6 -->
  <testcase classname="TestCases.login" name="redirection_dashboard" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy30.isDisplayed(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:304)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:290)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:287)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:203)
at pages.LoginPage.checkLogin(LoginPage.java:147)
at TestCases.login.redirection_dashboard(login.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- redirection_dashboard -->
</testsuite> <!-- Login -->
